pkg/tcpip/adapters/gonet/gonet_test.go

```diff
@@ -480,13 +480,14 @@ func TestUDPForwarder(t *testing.T) {
 	}
 
 	done := make(chan struct{})
-	fwd := udp.NewForwarder(s, func(r *udp.ForwarderRequest) {
+	fwd := udp.NewForwarder(s, func(r *udp.ForwarderRequest) bool {
 		defer close(done)
 
 		var wq waiter.Queue
 		ep, err := r.CreateEndpoint(&wq)
 		if err != nil {
 			t.Fatalf("r.CreateEndpoint() = %v", err)
+			return false
```

Comment:
Note that `Fatalf()` doesn't return, it will fail the test and runs `runtime.Goexit`.